client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/SparkShuffleManager.java

```diff
@@ -420,6 +424,20 @@ private int executorCores(SparkConf conf) {
     }
   }
 
+  private void checkUserClassPathFirst() {
```

Comment:
Do we need to implement similar check in spark-2 ?

Reply:
@cxzl25, `spark.executor.userClassPathFirst` config option is experimental in spark-2. Therefore, IMO, we does not need to follow the same check in spark-2. cc @mridulm, WDYT?

Reply:
`spark.executor.userClassPathFirst` is also marked as experimental in the master branch.

https://github.com/apache/spark/blob/master/docs/configuration.md?plain=1#L773-L779